optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/report/BenchmarkReport.java

```diff
@@ -498,8 +498,8 @@ private void writeAverageCalculateCountPerSecondSummaryChart() {
         }
         XYPlot plot = createScalabilityPlot(seriesList,
                 "Problem scale", NumberFormat.getInstance(locale),
-                "Average calculate count per second", NumberFormat.getInstance(locale));
-        JFreeChart chart = new JFreeChart("Average calculate count summary (higher is better)",
+                "Average calculation count per second", NumberFormat.getInstance(locale));
+        JFreeChart chart = new JFreeChart("Average calculation count summary (higher is better)",
```

Comment:
Some time ago, I renamed "average calculation count per second (999)" to "score calculation speed (999/sec)". Looks like I missed these - we should rename them also.